github/actions_workflow_jobs.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// Step represents a single task from a sequence of tasks of a job.
+type Step struct {
+	Name        string    `json:"name"`
+	Status      string    `json:"status"`
+	Conclusion  string    `json:"conclusion"`
+	Number      int64     `json:"number"`
+	StartedAt   Timestamp `json:"started_at"`
+	CompletedAt Timestamp `json:"completed_at"`
+}
+
+// Job represents a repository action workflow job.
+type Job struct {
+	ID          int64     `json:"id"`
+	RunID       int64     `json:"run_id"`
+	RunURL      string    `json:"run_url"`
+	NodeID      string    `json:"node_id"`
+	HeadSHA     string    `json:"head_sha"`
+	URL         string    `json:"url"`
+	HTMLURL     string    `json:"html_url"`
+	Status      string    `json:"status"`
+	Conclusion  string    `json:"conclusion"`
+	StartedAt   Timestamp `json:"started_at"`
+	CompletedAt Timestamp `json:"completed_at"`
+	Name        string    `json:"name"`
+	Steps       []*Step   `json:"steps"`
```

Comment:
`Steps` would be the only exception and is fine as it is currently written, being a slice of pointers.